### ISSUE TITLE
Extracting Motion Vector

### ISSUE THREAD

**Issue by zara6697** [LukasBommes/mv-extractor#26]
Dear all,
Thanks for your interesting implementation. There is a question, Is the motion vector extracted after the decoding process?
Best regards

**Comment by isra60** [comment]
Yes it is. motion vector are extracted after complete decoding process. It is necessary since the encoder only contains the Motion difference (MVD) and not the complete vector source and destiny. Also the motion difference coding is dependent of CABAC or CAVLC
